app/src/main/java/com/appunite/loudius/ui/components/LoudiusFullScreenError.kt

```diff
@@ -19,31 +19,92 @@ package com.appunite.loudius.ui.components
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.appunite.loudius.R
 import com.appunite.loudius.ui.theme.LoudiusTheme
 
+private const val minHeightToFitSpacer = 1856
+
 @Composable
 fun LoudiusFullScreenError(
     modifier: Modifier = Modifier,
     errorText: String = stringResource(id = R.string.error_dialog_text),
     buttonText: String = stringResource(id = R.string.try_again),
     onButtonClick: () -> Unit,
+) {
+    val density = LocalDensity.current
+    val configuration = LocalConfiguration.current
+    val screenHeight = with(density) { configuration.screenHeightDp.dp.roundToPx() }
```

Comment:
question: have you considered comparing values in dp instead of px? I think it could be a safer solution. You can see the sample sizes here https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes#window_size_classes

Reply:
I can't take values from this article because the smallest screen is in `medium height`. I can only do it in dp instead of px, but with this values.